include/lorawan/lorawan.h

```diff
@@ -103,6 +104,35 @@ struct lorawan_join_config {
 	enum lorawan_act_type mode;
 };
 
+#define LW_RECV_PORT_ANY UINT16_MAX
+
+struct lorawan_downlink_cb {
+	/* Port to handle messages for:
```

Comment:
Can you please move the comments inside this struct outside? It is okay to place small comments inside the struct but for the documentation of the fields, it is preferred to be done outside.

Reply:
I would prefer to leave it as is if possible. Moving the documentation to the top of the struct makes intellisense less useful IMO. This style is also used elsewhere in-tree, in Bluetooth for example:
https://github.com/zephyrproject-rtos/zephyr/blob/bfb1634af595207aa98e4a7233d12edb9b6769d7/include/bluetooth/bluetooth.h#L75-L88
It seems every subsystem has its own style for this however, so I will change it if you disagree

Reply:
Coming from the Linux kernel background I'd like to see the doc outside of the struct. But in this specific case, the doc is more elaborate as it explains the fields, etc... so let's keep it as it is.